### ISSUE TITLE
Exact search for project name in advanced process/task search

### ISSUE THREAD

**Issue by mnscholz** [kitodo/kitodo-production#5473]
**Describe the bug**
It does not seem to be possible to search for the exact name of the project in the advanced process/task search.
A search via the project search field seems to do a "contains" or "starts with" comparison: If you have two projects where the name of the first is a prefix of the name of the second one, a search for the first also generates results for the second project. 

Although the comparator reads  "contains"/"enthält", the search field is a dropdown where you can only choose from the available projects. That's why I would expect it to only find exact matches and therefore file this a bug. 

**To Reproduce**
1. Create a project called "ABC"
2. Create a project called "ABC DEF"
3. Create a process for each project
4. Go to the advanced process search form and select project "ABC" for the project search field. Search for for processes
5. The result list will also show the process belonging to project "ABC DEF"

**Expected behavior**
The result list should list process of project "ABC" only.

**Release**
3.4.1

**Desktop (please complete the following information):**
Firefox 107 on Windows 10

**Comment by henning-gerhardt** [comment]
For me this is not a bug, it is a maybe not well documented feature from Kitodo.Production 2.x and it is working like expected. Maybe we need here an other drop down value apart from "contains" and "contains not" to reflect your more feature request.

**Comment by henning-gerhardt** [comment]
You are not limited to the given list entries! After you done one search you can customize the search and adjust it to your needs and store this custom store for later use. Your given search should be work by "project: ab" to get all processes of projects with beginning "ab" even if this project is not direct available.

Maybe an other (strong) improvement should be that this lists are suggestions so you can choose one of them or enter your need.

**Comment by mnscholz** [comment]
You are right. However, customisation is (afaics) only possible in the single search slot or directly in the url by using the `:` notation. The presentation as a closed dropdown in the advanced search form remains at least irritating (if not a bug) for me.

Your suggestion to either choose from existing values or type in own search words would remedy this inconsistency. (y)

**Comment by mnscholz** [comment]
Should I write a separate feature request for implementing exact search for project field?
